test/e2e/predictor/test_autoscaling.py

```diff
@@ -160,7 +160,7 @@ def test_sklearn_kserve_cpu():
     kserve_client.delete(service_name, KSERVE_TEST_NAMESPACE)
 
 
-@pytest.mark.fast
+@pytest.mark.slow
```

Comment:
We can separate raw deployment tests into its own test suite

Reply:
Yes @yuzisun .. I added new e2e test group called raw and added all raw deployment tests.